### ISSUE TITLE
Scheduler - grouping by date

### ISSUE THREAD

**Issue by dxbykov** [DevExpress/DevExtreme#5372]
# The Problem
Currently, the Scheduler does not allow you to view appointments grouped by date if they are already grouped by resource. For instance, when you are planning a workday for several employees:

![image](https://user-images.githubusercontent.com/8307562/45363339-ef2f7780-b5df-11e8-8630-4cdca72f7632.png)

# The Proposed Solution

We introduced a new boolean option called `groupByDate`. This option can be specified for individual views.

## Group appointments by date, then by resource

The code sample below illustrates grouping by date and then by resource (as in the previous image).

```js
{
  type: "workWeek",
  groupByDate: true,
  ...
}
```

## Group appointments by resource, then by date

```js
{
  type: "workWeek",
  groupByDate: false, // default value
  ...
}
```

# We Need Your Feedback

## Take a Quick Poll

[Does the new appointment grouping type meet your needs?](https://docs.google.com/forms/d/e/1FAIpQLSfpw7ukg9cgk6Ghc1V6EZc3M3GjehsGHcZ_GlcWycblKlvsWw/viewform)

## Live Sandbox

[jQuery](https://codepen.io/anon/pen/aRYxvG)

# Get Notified of Updates

Subscribe to this thread or our [Facebook](https://www.facebook.com/DevExpress.DevExtreme/) and [Twitter](https://twitter.com/devextreme) accounts for updates on this topic.

**Comment by dxbykov** [comment]
Thanks to everybody who gave us early feedback on this feature. The feature is now available in the [v18.2](https://github.com/DevExpress/DevExtreme/releases/tag/18.2.3) release. I'm closing this thread. In the case of bugs or questions, feel free to create new GitHub [issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).

**Comment by godrage** [comment]
I would like to add that for my projects we would **require** vertical resource grouping and group-by-date.

the typical scenario for this would be the default workweek view with resources being staff.
so vertical staff-grouped-by-date is in our opinion the only efficient way to display a weeks schedule for a department and is currently not doable.
